docs/development-environment.md

```diff
@@ -323,39 +323,57 @@ To get started, there are several ways to run the unit tests, depending on how y
 
 ## JavaScript unit tests
 
-Jetpack includes also several [Jest](https://jestjs.io/) based unit tests.
-To execute them in your local environment, you can use the following commands.
+The `jetpack test` command can be run from the monorepo's root to run a specific project's tests.
 
-* ### Admin Page unit tests
+This may be of limited benefit locally during development since it isn't possible to use watch mode or run tests only for an individual file.
 
-	Standing on your jetpack directory, run
+Each project within the monorepo may also have its own test commands, so an alternative is to `cd` into the project's root, and run the test commands from there.
 
-	```sh
-	pnpm install
-	pnpm test-client
-	pnpm test-gui
-	```
+### Packages
 
-* ### Jetpack modules unit tests
+Packages may have a package.json in the root that has a `scripts` entry, and this details the different types of test commands that can be run, `pnpm run test` is the usual command for JavaScript unit tests.
 
-	Standing on your jetpack directory, run
+For example, to run an individual test file in watch mode:
+```sh
+cd projects/packages/forms
+pnpm run test --watch -- path/to/test/file.js
+```
 
-	```sh
-	pnpm install
-	pnpm test-modules
-	```
+### Jetpack Plugin
 
-	You can also only run tests matching a specific pattern. To do that, use the argument `-g, --grep <pattern>`:
+The Jetpack plugin project also has some additional test commands that can be run from its root.
 
-	```sh
-	pnpm test-gui -g 'my custom pattern to filter tests'
-	```
+### Admin Page unit tests
 
-	To use a custom reporter, pass the argument `-R, --reporter <name>`:
+TODO - explain the purpose of these tests.
 
-	```sh
-	pnpm test-client -R 'my_reporter'
-	```
+```sh
+cd projects/plugins/jetpack
+pnpm test-adminpage
+```
+
+This runs both the `client` and `gui` tests, but they can also be run individually using `pnpm test-gui` or `pnpm test-client`.
+
+You can also only run tests matching a specific pattern. To do that, use the argument `-g, --grep <pattern>`:
+
+```sh
+pnpm test-gui -g 'my custom pattern to filter tests'
+```
+
+To use a custom reporter, pass the argument `-R, --reporter <name>`:
+
+```sh
+pnpm test-client -R 'my_reporter'
+```
+
+### Extension unit tests
+
+TODO - explain the purpose of these tests.
```

Comment:
Ditto here, a brief explainer would be good (if anyone can help with that 🙏  )

Reply:
This is all about the editor. Apparently "editor extensions" is the term for "blocks, sidebars, and other stuff".